echo-notifications/src/main/groovy/com/netflix/spinnaker/echo/config/RetrofitConfig.groovy

```diff
@@ -1,43 +0,0 @@
-/*
- * Copyright 2015 Netflix, Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *    http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-
-
-package com.netflix.spinnaker.echo.config
-
-import groovy.transform.CompileStatic
-import groovy.util.logging.Slf4j
-import org.springframework.context.annotation.Bean
-import org.springframework.context.annotation.Configuration
-import retrofit.RestAdapter
-import retrofit.client.Client
-import retrofit.client.OkClient
-
-@Configuration
-@Slf4j
-@CompileStatic
-class RetrofitConfig {
-
-    @Bean
```

Comment:
Since we removed a type of this bean and we haven't changed any places where we refer this, not sure how the wiring works now ..?